.gitignore

```diff
@@ -20,12 +20,14 @@ TAGS
 .history
 .idea
 
-.vagrant
+.vagrant/
+.bashrc
```

Comment:
I think this is a holdover from when I was mounting the project root at `/home/vagrant` instead of `/home/vagrant/geotrellis-site`.